<ui version="4.0" >
 <class>DlgMidiLearning</class>
 <widget class="QDialog" name="DlgMidiLearning" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MIDI Learning Wizard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="label_2" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Welcome to the MIDI Learning Wizard</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>This wizard allows you to easily map the controls on your MIDI controller to Mixxx's controls.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>116</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="pushButtonBegin" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Begin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>116</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Please tweak the control for:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QLabel" name="labelMixxxControl" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Mixxx Control</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QLabel" name="labelMappedTo" >
         <property name="text" >
          <string>Successfully mapped to: </string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QPushButton" name="pushButtonPrevious" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QPushButton" name="pushButtonSkip" >
         <property name="text" >
          <string>Skip</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Press spacebar to proceed or skip.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <widget class="QLabel" name="label_4" >
       <property name="geometry" >
        <rect>
         <x>40</x>
         <y>20</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>MIDI learning complete!</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>60</y>
         <width>281</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This wizard can be re-run at any time.&lt;/p>
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;br />Now go spin some beats!&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonFinish" >
       <property name="geometry" >
        <rect>
         <x>110</x>
         <y>200</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text" >
        <string>Finito!</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
